### DISCUSSION TITLE
Timestamps automatic

### DISCUSSION THREAD

**Discussion by gutyerrez** [laravel/framework#33867] [General] [answered]
My sugestion is to make the `$table->timestamps();` insert default value, to `crated_at` and `updated_at` not only in code, but in sql table too, currently only insert timestamps value when i insert from code, change this to sql table auto execute.

**Comment by nandi95** [comment, marked as answer]
There's even methods for these
https://laravel.com/api/9.x/Illuminate/Database/Schema/ColumnDefinition.html#method_useCurrent
Has this not been done because of database engine differences?
